ci-operator/config/openshift/grafana-tempo-operator/openshift-grafana-tempo-operator-main.yaml

```diff
@@ -1,10 +1,34 @@
+base_images:
+  base:
+    name: "4.14"
+    namespace: ocp
+    tag: base
+  ocp_builder_rhel-9-golang-1.20-openshift-4.14:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.20-openshift-4.14
 binary_build_commands: go install ./
```

Comment:
Do we need to run make build here https://github.com/grafana/tempo-operator/blob/main/Makefile#L131 instead of go install command.